apis/projectcontour/v1/httpproxy.go

```diff
@@ -119,6 +119,9 @@ type TLS struct {
 	// backing cluster.
 	// +optional
 	Passthrough bool `json:"passthrough,omitempty"`
+	// ClientValidation defines how to verify the client certificate
```

Comment:
The audience for this comment are developers and operators trying to understand how to use the API, so we can err on the side of verbosity here. This setting

1. Enables TLS client certificate validation
1. Requires clients to present a TLS certificate (i.e. not optional validation)
1. Specifies how the client certificate will be validated

Do you think that covers it?

Reply:
Yes, that it good description.
I'm not sure if there should be some markup to get HTML list rendered correctly?